src/features/session_replay/aggregate/index.js

```diff
@@ -113,6 +113,9 @@ export class Aggregate extends AggregateBase {
     /** Hold on to the last meta node, so that it can be re-inserted if the meta and snapshot nodes are broken up due to harvesting */
     this.lastMeta = undefined
 
+    /** set by BCS response */
+    this.entitled = false
+
     const shouldSetup = (
       getConfigurationValue(agentIdentifier, 'privacy.cookies_enabled') === true &&
       getConfigurationValue(agentIdentifier, 'session_trace.enabled') === true
```

Comment:
Should this be `session_replay.enabled`?

Reply:
This is a modest attempt to not run sr if trace is not going to run since every replay needs a trace